packages/auth/amplify_auth_cognito/lib/src/CognitoSignIn/CognitoSignInOptions.dart

```diff
@@ -17,10 +17,13 @@ import 'package:amplify_core/amplify_core.dart';
 
 class CognitoSignInOptions extends SignInOptions {
   Map<String, String>? clientMetadata;
-  CognitoSignInOptions({this.clientMetadata}) : super();
+  final AuthenticationFlowType? authFlowType;
```

Comment:
How about just `authFlow`. The `type` suffix originates in the SDK where they add `Type` to the end of all types for some reason

Reply:
Used `AuthenticationFlowType` because it matches what is in the config, which is what get's utilized as the default. Even though it's a little more verbose, I think having that consistency is helpful.

Reply:
The API parameter `authFlowType` is what is used by the native libraries, so I decided to stay consistent with them. In terms of the enum `AuthenticationFlowType`, I used the existing one we are getting from the config. I realize it's a little odd to have a mismatch between the type name and the API parameter, but I didn't want to deviate from the native APIs or change/duplicate an existing enum.